### PR TITLE
Sun Animation

### LINKED ISSUE #89: Baby Sun Animation
**Is your feature request related to a problem? Please describe.**
Make the baby sun animation

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Create sun animation

## Purpose
Enhance UX

## Approach
Use keyframes to adjust the position of the sun image

Closes #89

### CODE REVIEW

File: the-toad-tribune/src/layout/Weather.tsx [hunk 1]
```diff
@@ -28,6 +28,18 @@ const Weather: React.FC<IWeatherProp> = ({ darkMode, weatherResponse }) => {
...
+       transform: translate(0px, 0px);
+       animation-timing-function:ease-in-out
+   }
+
+   100% {
+       transform: translate(0px, -180px);
+       animation-timing-function:ease-in-out
+   }
+`
+
```
Comment (JorgeLVilla) [inline the-toad-tribune/src/layout/Weather.tsx:42]: Teletubbies vibes...

**Review by connielion — APPROVED** [review]
LGTM!

**Review by JorgeLVilla — APPROVED** [review]
Noice!

File: the-toad-tribune/src/layout/Weather.tsx [hunk 2]
```diff
@@ -28,6 +28,18 @@ const Weather: React.FC<IWeatherProp> = ({ darkMode, weatherResponse }) => {
 
 export default Weather;
 
+const babySunAnimation = keyframes`
```
Comment (Gjhnsn) [inline the-toad-tribune/src/layout/Weather.tsx:31]: 🌞

**Review by Gjhnsn — APPROVED** [review]

File: the-toad-tribune/src/layout/Weather.tsx (hunk 1 above)
Comment (dannysanchez559) [reply the-toad-tribune/src/layout/Weather.tsx:42]: babyyyyyy

**Review by dannysanchez559 — COMMENTED** [review]
looks fine

**Pull request merged**